### PR TITLE
feat: introduce enableOutboundRelay method

### PR DESCRIPTION
**Pull request by jeremywagemans**
# Why
The SDK used to retrieve the outbound relay configurations from the API every 2 minutes which was causing some friction when integrating with Outbound Relay and regularly updating the configurations in the dashboard.
There was also an issue where requests weren't routed to O.R. straight after enabling it with the EV client.

# How
We've implemented the following changes:
- Add a default poll interval of 5 seconds
- Dynamically updating the poll interval based on the server configuration (`X-Poll-Interval` response header)
- Introduced a new async method that allows SDK users to enable outbound relay

### CODE REVIEW

File: lib/utils/httpsHelper.js [hunk 1]
```diff
@@ -63,7 +63,10 @@
   function wrapMethodRequest(...args) {
     const domain = getDomainFromArgs(args);
     const shouldProxy = domainFilter(domain);
-    if (debugRequests) {
+    if (
+      debugRequests &&
+      !(domain.endsWith('evervault.com') || domain.endsWith('evervault.io'))
```
[the remark this answers is not in the document]
Comment (jeremywagemans) [reply lib/utils/httpsHelper.js:68]: False positive

File: lib/core/relayOutboundConfig.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+const clearCache = () => {
+  decryptionDomainsCache = null;
+};
+
+const getDecryptionDomains = () => {
+  return decryptionDomainsCache;
+};
+
+const init = async (config, http) => {
+  var pollingInterval = config.http.pollInterval;
```
Comment (eoinboylan) [inline lib/core/relayOutboundConfig.js:29]:
```suggestion
  let pollingInterval = config.http.pollInterval;
```

File: lib/index.js [hunk 3]
```diff
@@ -392,6 +381,39 @@ class EvervaultClient {
     return response.body;
   }
 
+  async enableOutboundRelay(options = {}) {
+    validationHelper.validateRelayOutboundOptions(options);
+    if (!options || !options.decryptionDomains) {
+      var debug_request;
```
Comment (eoinboylan) [inline lib/index.js:387]:
```suggestion
      let debug_request;
```

**Review by eoinboylan — APPROVED** [review]

**Review by eoinboylan — APPROVED** [review]

**Pull request merged**